src/LibChorus/FileTypeHandlers/lift/LiftUtils.cs

```diff
@@ -6,7 +6,9 @@ namespace Chorus.FileTypeHandlers.lift
 {
 	public static class LiftUtils
 	{
+		[Obsolete("Use LiftTimeFormatWithTimeZone instead, as LiftTimeFormatNoTimeZone produces incorrect results when used with DateTime.Now")]
 		static public string LiftTimeFormatNoTimeZone = "yyyy-MM-ddTHH:mm:ssZ";
+		static public string LiftTimeFormatWithTimeZone = "yyyy-MM-ddTHH:mm:ssK";
```

Comment:
I know this was public, but it looks like there are no examples of use in our github organization. So I would prefer to keep the code clean and just rename without adding the Obsolete comment. And since noone is using it yet we may as well make it internal

Reply:
I'll certainly just clean up the `LiftTimeFormatNoTimeZone` and related constants rather than make them obsolete. OTOH, I'm not a huge fan of making this internal: what do we gain by hiding the time format string? We don't gain the ability to change it: changing it would be a breaking change whether or not it's part of the public-facing API, because it would change the format of the XML produced by Chorus. And since this is used from other namespaces (e.g., XmlUtilities.AddDateCreatedAttribute uses LiftUtils.LiftTimeFormatWithTimeZone), if we were ever to split the Lift and XML-handling parts of Chorus into two separate DLLs, making LiftTimeFormatWithTimeZone internal would just complicate that process.

If you feel that this really *should* be internal I'll go ahead and make that change, but I'll leave it public for now since I believe that's the way cause the fewest possible problems for ourselves in the future.